sycl/test-e2e/lit.cfg.py

```diff
@@ -720,6 +718,10 @@ def open_check_file(file_name):
             # str.removeprefix isn't universally available...
             sg_sizes_str = line.strip().replace("info::device::sub_group_sizes: ", "")
             dev_sg_sizes.append(sg_sizes_str.strip().split(" "))
+        if re.match(r" *DeviceID*", line):
+            _, device_id = line.strip().split(":", 1)
+            if device_id.strip() == "3034":             
```

Comment:
Can we add a variable for "3034"? Something like `gpu-intel-pvc-1T-device-id`? Otherwise it just seems a magic number here.

Reply:
Sure, can do.